.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: java
+
+jdk:
+  - openjdk7
+  - oraclejdk7
```

Comment:
Good catch. Set it to 1.7 (even though originally it was 1.8). Let's see if Travis build succeeds.